### PR TITLE
feat: create profile bio

### PR DESCRIPTION
**Pull request by Simpsonkk**
## Describe your changes

- I did a profile bio step.
- Initial view (screenshot 1).
- When users upload video formats mp4 or mov, I show them a preview of the video (screenshot 2). 
- When users upload avi format,  I show the icon instead of the video preview because the tag 'video' doesn`t support the avi format (screenshot 3).
- When users upload other formats I show an error message and error icon (screenshot 4).
- Show an error if the video size exceeds 50 MB (screenshot 5).

screenshot 1
image

screenshot 2
image

screenshot 3
image

screenshot 4
image

screenshot 5
image

## Issue ticket code (and/or) and link

- [Link to JIRA ticket](https://homecareaid.atlassian.net/browse/AC-74?atlOrigin=eyJpIjoiN2I1NjFiYWE5MjZjNGIzMThhMjE0MzRkZDY5ZjkwYzkiLCJwIjoiaiJ9)

### **General**

- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [ ] Updated the documentation
- [x] Performed a self-review of my code
- [x] Types for input and output parameters
- [x] Don't have "any" on my code
- [ ] Used the try/catch pattern for error handling
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside the ternary operator
- [x] Don't have commented code
- [x] no links in the code, env links should be in env file (for example: server url), constant links (for example default avatar URL) should be in constant file.
- [x] Used camelCase for variables and functions
- [x] Date and time formats are on the constants
- [x] Functions are public only if it's used outside the class
- [x] No hardcoded values
- [ ] covered by tests
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Frontend

- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the constants
- [x] No text in the components, use i18n approach
- [x] No inline styles
- [x] Imports are absolute
- [x] Attach a screenshot if PR has visual changes.

### CODE REVIEW

**Review by nadiia-tsytsylina — APPROVED** [review]

**Review by ProggerOleg — APPROVED** [review]

**Review by fedasevich — APPROVED** [review]

File: src/locales/langs/en.ts [hunk 1]
```diff
@@ -157,6 +157,65 @@ const en = {
...
+
+    qualification: 'Qualification',
+    workExperience: 'Work Experience',
+    services: 'Services',
+    availability: 'Availability',
+    rates: 'Rates',
+    bio: 'Bio',
+  },
+  profileQualification: {
+    certificationNameMaxLength: 'Certification name must be at most 100 characters',
```
Comment (Lilly-nn) [inline src/locales/langs/en.ts:172]: it would be better to get the max amount of characters from constansts

File: src/locales/langs/en.ts [hunk 2]
```diff
@@ -157,6 +157,65 @@ const en = {
...
+    certificationNameRequired: 'Certification name is required',
+
+    certificationNumberMaxLength: 'Certification number must be at most 100 characters',
```
Comment (Lilly-nn) [inline src/locales/langs/en.ts:175]: here too and so on

Comment (Simpsonkk) [reply src/locales/langs/en.ts]: done

File: src/locales/langs/en.ts (hunk 1 above)
Comment (Simpsonkk) [reply src/locales/langs/en.ts]: done

**Review by Lilly-nn — APPROVED** [review]

File: src/components/profile/profile-qualification/certificate-form/constants.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { ProfileQualityFormValues } from 'src/components/profile/profile-qualification/types';
+
+export const DATE_FORMAT = 'dd/MM/yyyy';
+
+export const URL_PATTERN = /^https:\/\/.+$/;
```
Comment (lyudad) [inline src/components/profile/profile-qualification/certificate-form/constants.ts:5]: hmmm I already seen this code. you don't make your branch from develop branch/ each PR should be independent in other case it does to make sense in branching

Comment (Simpsonkk) [reply src/components/profile/profile-qualification/certificate-form/constants.ts]: fixed

**Review by lyudad — APPROVED** [review]

**Pull request merged**